tests/testutil/testutil.go

```diff
@@ -24,11 +25,12 @@ func init() {
 }
 
 // Ensure CLI flag is passed, this way integration tests won't run during unit tests
-func PreCheck() {
-	runTests := flag.Bool("integration-tests", false, "must be provided to run the integration tests")
+func PreCheck(flagToCheck string) {
```

Comment:
Passing in string like this is brittle, need to fix if we continue down the dual test path.

Reply:
Would it be better to have two functions here? `PreCheckIntegration` and `PreCheckValidation`?

Reply:
Yea I guess, meh, feels gross either way. Or make them private methods in the suite files themselves.

Reply:
Should make them const, or separate methods. Don't like the string still.